debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlValueConverters.java

```diff
@@ -599,29 +565,21 @@ protected Object convertPoint(Column column, Field fieldDefn, Object data){
      * @throws IllegalArgumentException if the value could not be converted but the column does not allow nulls
      */
     protected Object convertGeometry(Column column, Field fieldDefn, Object data) {
-        if (data == null) {
-            data = fieldDefn.schema().defaultValue();
-        }
-
-        Schema schema = fieldDefn.schema();
-
-        if (data == null) {
-            if (column.isOptional()) {
-                return null;
+        return convertValue(column, fieldDefn, data, (Supplier<?>)() -> {
```

Comment:
I still struggle to understand why the supplier is needed. Why not simply pass the "empty" `Point` instance?

Reply:
@gunnarmorling You are (almost) right, I can create the empty instance in advance. I know that the value is not optional and schema has no default value so I can pre-bake it in advance. But, IIRC columnmapper can modify the column schema and we will lose the modification then, would not we?